src/scry/text_document.cr

```diff
@@ -61,7 +61,7 @@ module Scry
     end
 
     def self.uri_to_filename(uri)
-      uri.sub(/^file:\/\/|^inmemory:\/\/|^git:\/\/|^untitled:/, "")
```

Comment:
All the LSP is specified in the `src/protocol` folder. Files outside aren't forced to follow it but to implement features. I guess the `inmemory:` and `git:` are needed to avoid errors in diff views inside VSCode. I removed the untitled part because wasn't needed here.

Seems atom just use a plain word `untitled` for Untitled files, so if atom/sublime/other support formatting untitled files, we should open a new PR and add support for it :+1: 

These are the lines we should modify:

https://github.com/crystal-lang-tools/scry/blob/e0a5698eb0689caf8438f6884fed99a3da351e04/src/scry/protocol/diagnostic.cr#L28

and this

https://github.com/crystal-lang-tools/scry/blob/07cb031d75ed778a106dd89d4ab8ceb0e3b213d7/src/scry/text_document.cr#L72

Changing `untitled:` by something like `untitled` or check other words used in sublime/vim/other

Reply:
@laginha87 I gonna merge it, I just opened a tracking issue about this, see: https://github.com/crystal-lang-tools/scry/issues/87 :+1: